pregred-gui/src/app/components/app-footer/AppFooter.css

```diff
@@ -0,0 +1,29 @@
+.footer-container {
+  bottom: 0;
+  width: 100%;
+  background-color: #000000;
```

Comment:
Can you make this bg colour similar to the navbar background colour?. its a variety of dark blue.